wiki/11111.md

```diff
@@ -0,0 +1,294 @@
+# Curl Examples to front-end
```

Comment:
Is this file a duplication of Front-end_with_curl.md? That one has 287 rows though and this one has 294.

Reply:
Done.